# robot_params.yaml
wheel_odometry_node:
  ros__parameters:
    wheel_radius: 0.058  # Meters
    wheel_base: 0.284     # Meters
    # Add other parameters as needed

motor_command_node:
  ros__parameters:
    wheel_radius: 0.058  # Meters
    wheel_base: 0.284     # Meters
